docusaurus/video/docusaurus/docs/api/call_types/manage-types.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: manage_types
+sidebar_position: 2
+slug: /call_types/manage
+title: Manage Types
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Read call types
+
+<Tabs groupId="examples">
+<TabItem value="js" label="JavaScript">
+
+```js
+client.getCallTypes();
+
+//or
+client.getCallType('livestream');
+```
+
+</TabItem>
+</Tabs>
+
+## Create call type
+
+<Tabs groupId="examples">
+<TabItem value="js" label="JavaScript">
+
+```js
+client.createCallType({
+  name: 'allhands',
+  settings: {
+    audio: { mic_default_on: true, default_device: 'speaker' },
+  },
+  grants: {
```

Comment:
I know the `grants` field works syntactically, but does this example make sense semantically?